app/src/main/java/com/ciandt/dragonfly/example/features/feedback/FeedbackActivity.kt

```diff
@@ -81,9 +91,11 @@ class FeedbackActivity : BaseActivity(), FeedbackContract.View {
                 saveImageToGalleryInteractor,
                 benchmarkInteractor)
 
-        presenter.attachView(this)
-        presenter.setUserFeedback(userFeedback)
-        presenter.setClassifications(classifications)
+        presenter.apply {
+            attachView(this@FeedbackActivity)
+            setUserFeedbacks(userFeedbacks)
```

Comment:
setUserFeedback (feedback is an uncountable noun: https://www.quora.com/What-is-the-plural-of-feedback-Can-you-ever-use-feedbacks)